libyul/AsmCodeGen.cpp

```diff
@@ -48,10 +48,10 @@
 using namespace std;
 using namespace dev;
 using namespace langutil;
+using namespace yul;
 using namespace dev::solidity;
```

Comment:
This should go away at some point. No mention of `solidity` inside libyul.

Reply:
agreed. it's one of the few left-overs, that i'd like to tackle at a later point in a seperate PR.